instrumentation/afl-compiler-rt.o.c

```diff
@@ -1978,6 +2003,43 @@ void __sanitizer_cov_trace_pc_guard_init(uint32_t *start, uint32_t *stop) {
 
 #endif  // __AFL_CODE_COVERAGE
 
+  // IJON SUPPORT: Expand map size when AFL_IJON=1 to ensure AFL tools
+  // read the full map that will contain IJON coverage points
+  if (getenv("AFL_IJON")) {
+    // CONDITIONAL EXPANSION: Different logic for ≤65k vs >65k maps
```

Comment:
thinking about this, this will not work.
pc_guard_init can be called multiple times during startup, and this will then break.
This should rather be moved into the starting section of the forkserver, WDYT?